env/includes/settings/tk-multi-createdelivery2.yml

```diff
@@ -0,0 +1,151 @@
+  # import
+  # Note: multi-entity type fields are not supported
+  version_field_columns: [entity, sg_version_type, description]
+
+  # List of templates to use when attempting to parse tokens from the version name,
+  # such as for the version number or linked entity. The order is important, the
+  # first matching template is used.
+  version_name_templates:
+  - shot_version_name
+  - shot_version_name_generic
+  - asset_version_name
+  - version_name_generic
+
+  # A template defining the destination path for deliveries. Not used by all
+  # delivery transfer modes.
+  delivery_root_path_template: delivery_root_path
+
+  # Default value to define who the delivery will be "sent" to (updates addressings_to).
+  delivery_to: ["VFX Production"]
+
+  # Default value for the delivery type
+  delivery_type: Vendor Submission
+
+  # When set to true, checksums will be generated for all files to be delivered
+  # Note: this can be a time-consuming process, so the preference may be to set
+  # this to false
+  enable_delivery_checksums: true
+
+  # Ordered list of dictionaries of tokens used to detect different formats
+  #
+  # Note: the order of the dictionaries will determine which match is attempted
+  # first. A single file will only be matched to one format. If no suitable match
+  # is found, the format is detected based on matching
+  # tk-framework-delivery:delivery_formats:extensions
+  #
+  # format_name: Format from those defined in tk-framework-delivery:delivery_formats:name
+  #              which will be used when the corresponding tokens are matched
+  # token_type: Whether the token is a directory or template key. Possible values
+  #             are 'directory' or `template_key`
+  # token: A string to match in the path to the file, or a template to extract a token
+  #        from the filename.
+
+  format_token_mappings:
+  - format_name: editorial
+    token_type: directory
+    token: dnxhd
+  - format_name: h264
+    token_type: template_key
+    token: h264_token
+  - format_name: matte
+    token_type: template_key
+    token: matte_token
+
+  # An SG Project field name holding the secret key for the Aspera Files client
+  sg_aspera_files_secret_field: sg_aspera_files_cd_secret
+
+  # Aspera Files client public id
+  aspera_files_client_id: BpoPR5gMJw
+
+  # An SG Project field name holding the secret key for the Box client
+  # Default: sg_box_secret_field: sg_box_create_delivery_secret
+
+  # Validation hooks: allow specific formats to be validated. Check validation
+  # hooks docstrings for their settings.
+#  validation_hooks:
+#  - name: Movie Checker
+#    hook: "{self}/validate_movie.py"
+#    settings:
+#        fps: 23.98
+#        resolution: "1280x720"
+#        codec: "prores"
+#        bit_depth: 10
+#        show_value_columns: false
+
+# Create delivery2 for vendors: re-use settings above but overrides a couple of
+# them
+settings.tk-multi-createdelivery2.vendor:
```

Comment:
sorry just caught this... should this be `settings.tk-multi-createdelivery2-vendor` (`-vendor` instead of `.vendor`)? This is what matches our gpltstudio config.